src/test/rustdoc-ui/check-doc-attrs.rs

```diff
@@ -0,0 +1,18 @@
+#![doc(inline)]
+//~^ WARNING can only be applied to a `use` item
+
+#[doc(inline)]
+#[doc(no_inline)]
+//~^^ ERROR conflicting doc inlining attributes
+//~|  ERROR conflicting doc inlining attributes
+pub use bar::Bar;
+
+pub mod bar {
+    pub struct Bar;
+}
+
+#[doc(no_inline)]
+//~^ WARNING can only be applied to a `use` item
+#[doc(html_no_source)]
+//~^ WARNING can only be applied at the crate level
+fn main() {}
```

Comment:
nit: missing a newline at EOF, this will break `cat`.

Reply:
And CI 😕